.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,36 @@
+name: Bug report
+description: Report a bug.
+labels: ['type:bug', 'status:requirements', 'priority-5-triage']
+body:
+  - type: input
+    id: version-of-renovate-approve-bot-bitbucket-cloud-edition
+    attributes:
+      label: Version.
+      description: What version of the Renovate approve bot Bitbucket Cloud Edition are you using?
```

Comment:
This description can probably be trimmed down to just `approval bot`?

Reply:
As you like, I would suggest to always capitalize it: "Renovate Approve Bot"

https://www.thesaurus.com/e/grammar/when-to-capitalize-words/

Reply:
```suggestion
      description: What version of the Renovate Approve Bot are you using?
```